tokens/usecase/tokens_usecase_test.go

```diff
@@ -135,6 +142,21 @@ func (s *TokensUseCaseTestSuite) TestParseExponents_Testnet() {
 	s.Require().Equal(defaultCosmosExponent, osmoToken.Precision)
 }
 
+func (s *TokensUseCaseTestSuite) TestGetPrices_Coingecko() {
```

Comment:
Do you mind adding a short comment summary, specifying what this test aims to validate?

Reply:
Done